src/rgw/rgw_op.cc

```diff
@@ -1054,17 +1054,14 @@ void RGWGetObjTags::execute(optional_yield y)
   s->object->set_atomic(s->obj_ctx);
 
   op_ret = s->object->get_obj_attrs(s->obj_ctx, y, this);
-  if (op_ret < 0) {
-    ldpp_dout(this, 0) << "ERROR: failed to get obj attrs, obj=" << s->object
-        << " ret=" << op_ret << dendl;
-    return;
-  }
 
-  attrs = s->object->get_attrs();
-  auto tags = attrs.find(RGW_ATTR_TAGS);
-  if(tags != attrs.end()){
-    has_tags = true;
-    tags_bl.append(tags->second);
+  if (op_ret == 0){
+    attrs = s->object->get_attrs();
+    auto tags = attrs.find(RGW_ATTR_TAGS);
+    if(tags != attrs.end()){
+      has_tags = true;
+      tags_bl.append(tags->second);
+    }
```

Comment:
we can have the error log also in 'else' case, just dont return.

Reply:
Yeah it is logged in SAL layer